index.html

```diff
@@ -62,7 +63,7 @@ <h1 class="title">Emmanuel <br> Cortés</h1>
                     <li class="tech-btn">Ruby on Rails</li>
```

Comment:
- [OPTIONAL] Please fix the style of your "Start collaboration" button to be more responsive. Create a more professional look and a better UX and UI. As not required by the previous reviewer I consider it optional.
- [OPTIONAL] Your images were supposed to be background images defined on your CSS to avoid conflict with the text. Kindly fix the style to create a better UI. As not required by the previous reviewer I consider it optional.

<img width="852" alt="Screen Shot 2022-11-16 at 9 08 47 PM" src="https://user-images.githubusercontent.com/47673311/202323307-20081a54-1d64-4ab4-9f9a-3079685dccbe.png">